src/react/App.jsx

```diff
@@ -22,13 +22,63 @@ import AlertsProvider from './AlertManager';
 import Router from './Router';
 import Snackbar from './ui/Snackbar';
 
+function parseDashboardConfig(json) {
+  const {
+    astarte_api_url,
+    realm_management_url,
+    appengine_url,
+    pairing_url,
+    flow_url,
+    enable_flow_preview,
+    default_realm,
+    default_auth,
+    auth,
+  } = json;
+
+  if (!astarte_api_url || !auth || !default_auth) {
+    return null;
+  }
+
+  const config = {
+    astarte_api_url,
+    realm_management_url,
+    appengine_url,
+    pairing_url,
+    flow_url,
+    enable_flow_preview,
+    default_realm,
+    default_auth,
+    auth: {},
+  };
+
+  auth.forEach((authOption) => {
+    if (authOption.type === 'token') {
+      config.auth.token = {
+        enabled: true,
+      };
+    } else if (authOption.type === 'oauth') {
+      config.auth.oauth = {
+        enabled: true,
+        oauth_api_url: authOption.oauth_api_url,
```

Comment:
Maybe we could keep `oauth_api_url` as a camelCase property: `oauthApiUrl`

Reply:
All other properties use snake_case (for compability with the rest of the code base). It would be kind of strange having this part of the config as camel case